src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -0,0 +1,334 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Parameters returned by the WWW-Authenticate header. This allows for dynamic
+    /// scenarios such as claim challenge, CAE, CA auth context.
+    /// See https://aka.ms/msal-net/wwwAuthenticate.
```

Comment:
>See https://aka.ms/msal-net/wwwAuthenticate [](http://example.com/codeflow?start=8&length=43)

this page is great. would have liked some graphics showing the request/response flow

Reply:
we can add a graphic